galsim/image.py

```diff
@@ -317,7 +312,7 @@ def __init__(self, *args, **kwargs):
             try:
                 ncol = int(ncol)
                 nrow = int(nrow)
-            except Exception:
+            except:
```

Comment:
Actually, don't worry about this one.  I ended up addressing it on #799, since I was modifying other aspects of the Image constructor, so I fixed this too.